src/styles/blocks/catalog.scss

```
@@ -17,6 +18,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $time-smooth, color $time-smooth;
+
+    &:hover {
+      color: #34568b;
```

create variable for color
```suggestion
      color: $color;
```